controllers/volumereplicationgroup_controller.go

```diff
@@ -586,8 +681,21 @@ func (v *VRGInstance) processAsPrimary() (ctrl.Result, error) {
 		return ctrl.Result{Requeue: true}, nil
 	}
 
-	requeue := false
-	requeue = v.reconcileVRsAsPrimary()
+	if err := v.restorePVs(); err != nil {
```

Comment:
(from testing) require "create" role for PVs in kubebuilder directive above Reconcile function.